active/NumberBox/NumberBox.md

```diff
@@ -159,24 +185,31 @@ unsealed runtimeclass NumberBox : Windows.UI.Xaml.Controls.Control
     String Text; 
     String PlaceholderText;
```

Comment:
I also have Header and HeaderTemplate as properties.

Reply:
Added!